packages/driver/test/cypress/integration/commands/request_spec.coffee

```diff
@@ -57,6 +69,7 @@ describe "src/cy/commands/request", ->
             method: "GET"
             gzip: true
             followRedirect: true
+            timeout: 30000
```

Comment:
let's refactor this to instead of using a magic value - utilize `Cypress.config('responseTimeout')` so that it's clear where this comes from